git/repo.go

```diff
@@ -16,6 +16,7 @@ import (
 
 const (
 	DefaultCloneTimeout = 2 * time.Minute
+	CheckPushTag        = "flux-write-check"
```

Comment:
Does this need to be exported?

Reply:
As things stand, no, but since it's something we push to an external system, I thought it was better to expose it.